src/components/tasks/card/SuggestionBox.tsx

```diff
@@ -0,0 +1,103 @@
+import React, { FC, useEffect, useState } from 'react';
+import { DUMMY_PROFILE as placeholderImageURL } from '@/constants/display-sections';
+import classNames from '@/components/tasks/card/suggestion.module.scss';
+import fetch from '@/helperFunctions/fetch';
+import { userDataType } from '@/interfaces/user.type';
+import { GithubInfo } from '@/interfaces/suggestionBox.type';
+
+type Props = {
+    assigneeName: string;
+    showSuggestion: boolean;
+    onClickName: (userName: string) => void;
+};
+
+const SuggestionBox: FC<Props> = ({
+    assigneeName,
+    showSuggestion = false,
+    onClickName = () => undefined,
+}) => {
+    const [loading, setLoading] = useState<boolean>(false);
+    const [suggestions, setSuggestions] = useState<GithubInfo[]>([]);
+
+    const clickHandler = (e: React.MouseEvent<HTMLLIElement, MouseEvent>) => {
+        onClickName(e.currentTarget.innerText);
+    };
+
+    const fetchUsers = async () => {
+        setLoading(true);
```

Comment:
try catch kaha hai ?

Reply:
coming in the next push @vvaibhavdesai

Reply:
@vvaibhavdesai so the reason i did not add try catch was because i tested the conditions in which it may cause the error.
So the possible error causing scenario would be, if we are querying with an empty input value, in such case we get the error, that 'search input cannot be empty'. Now what i did was introduced 'if check on assigneeName', now if there is some value in assigneeName only then we make the request.
Now what i did was obviously a gross assumption, because there can be plenty of other reason for the request to return an error. so yes i did add a try catch in the fetch